dbos/cli.py

```diff
@@ -264,5 +270,68 @@ def migrate() -> None:
     typer.echo(f"Completed schema migration for database {app_db_name}")
 
 
+@app.command(help="Reset the DBOS system database")
+def reset(
+    yes: bool = typer.Option(False, "-y", "--yes", help="Skip confirmation prompt")
```

Comment:
I suggest we don't allow to skip the prompt

Reply:
This is standard to make it user to use programatically (e.g. for testing), pretty much every destructive action that requires a prompt (Terraform CLI, AWS CLI) has an option to skip the prompt.